tianshou/data/utils/converter.py

```diff
@@ -10,13 +10,17 @@ def to_numpy(x: Union[
     Batch, dict, list, tuple, np.ndarray, torch.Tensor]) -> Union[
         Batch, dict, list, tuple, np.ndarray, torch.Tensor]:
     """Return an object without torch.Tensor."""
-    if isinstance(x, torch.Tensor):
+    if isinstance(x, torch.Tensor):  # most often case
         x = x.detach().cpu().numpy()
+    elif isinstance(x, np.ndarray):  # second often case
+        pass
+    elif isinstance(x, (np.number, np.bool_, Number)) or x is None:
```

Comment:
It is a bit ugly to pass None to `np.asanyarray` knowing it is None in advance.

Reply:
But if x is None, the previous version of code also returns `np.asanyarray(None)`. I'm not sure whether returning `None` directly is okay.

Reply:
I don't know, but I find it more intelligible than `array(None, dtype=object)`.